components/admin-tabs/Applicants.tsx

```diff
@@ -187,44 +198,51 @@ const Applicants: React.FC = () => {
 
 const escaper = (s: string) => `"${s.replaceAll('"', '""')}"`;
 const separator = ',';
-const fileName = 'applicants.csv';
-const downloadFile = async (totalCount: number, filters: TableFilters, sorter: TableSorter) => {
+const fields: Array<keyof User> = ['email', 'isAdmin', 'applicationStatus', 'rsvpStatus'];
+type DownloadProps = {
+  totalCount: number;
+  filters: TableFilters;
+  sorter: TableSorter;
+};
```

Comment:
abstracted common args to download functions as a type. this is just for brevity–makes it so i dont have to separately pass each one in the `downloadApplicationCsv` and post acceptance functions, just the abstract download function.